sdk/src/index.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+export * from './suins-client.js';
```

Comment:
I don't feel too strongly about this, but I've been working on updating the the TS SDK exports to be explicit.  It makes things a LOT easier to reason about when refactoring when you have an explicit list of exports so you know whats being added/removed from the public API surface.  Using `export *` means a lot of things you don't intend for consumers of the SDK might be used and can't be changed in future versions

Reply:
Nice one, indeed helps with exporting only what might be needed externally! 👍